app/http.php

```diff
@@ -0,0 +1,1039 @@
+        $activeRuntimeId = $runtimeId; // Used with Swoole table (key)
+        $runtimeId = System::getHostname() . '-' . $runtimeId; // Used in Docker (name)
+
+        $runtimeHostname = \uniqid();
+
+        if ($activeRuntimes->exists($activeRuntimeId)) {
+            if ($activeRuntimes->get($activeRuntimeId)['status'] == 'pending') {
+                throw new \Exception('A runtime with the same ID is already being created. Attempt a execution soon.', 500);
+            }
+
+            throw new Exception('Runtime already exists.', 409);
+        }
+
+        $container = [];
+        $containerId = '';
+        $stdout = '';
+        $stderr = '';
+        $startTimeUnix = \microtime(true);
+        $endTimeUnix = 0;
+        $connection = $orchestrationPool->pop();
+        $orchestration = $connection->getResource();
+
+        $secret = \bin2hex(\random_bytes(16));
+
+        if (!$remove) {
+            $activeRuntimes->set($activeRuntimeId, [
+                'id' => $containerId,
+                'name' => $activeRuntimeId,
+                'hostname' => $runtimeHostname,
+                'created' => (int) $startTimeUnix,
+                'updated' => \time(),
+                'status' => 'pending',
+                'key' => $secret,
+            ]);
+        }
+
+        try {
+            Console::info('Building container : ' . $runtimeId);
+
+            /**
+             * Temporary file paths in the executor
+             */
+            $tmpSource = "/tmp/$runtimeId/src/code.tar.gz";
+            $tmpBuild = "/tmp/$runtimeId/builds/code.tar.gz";
+
+            /**
+             * Copy code files from source to a temporary location on the executor
+             */
+            $sourceDevice = getStorageDevice("/");
+            $localDevice = new Local();
+            $buffer = $sourceDevice->read($source);
+            if (!$localDevice->write($tmpSource, $buffer)) {
+                throw new Exception('Failed to copy source code to temporary directory', 500);
+            };
+
+            /**
+             * Create the mount folder
+             */
+            if (!\file_exists(\dirname($tmpBuild))) {
+                if (!@\mkdir(\dirname($tmpBuild), 0755, true)) {
+                    throw new Exception("Failed to create temporary directory", 500);
+                }
+            }
+
+            /**
+             * Create container
+             */
+            $variables = \array_merge($variables, [
+                'INTERNAL_RUNTIME_KEY' => $secret,
+                'INTERNAL_RUNTIME_ENTRYPOINT' => $entrypoint,
+                'INERNAL_EXECUTOR_HOSTNAME' => System::getHostname()
+            ]);
+            $variables = array_map(fn ($v) => strval($v), $variables);
+            $orchestration
```

Comment:
Would be great to get `cpu`,  `memory` and `swap` as API parameters.

Reply:
Should we remove these env variables all-together, or use them as max limit? Just like build max time?

Reply:
We can use them as the default values for the new params.

Reply:
- Add parameters for them
- No maximum.

Reply:
Introduced as parameters, removed limit env vars